schemas/jsonschema/PoseInFrame.json

```diff
@@ -0,0 +1,80 @@
+{
+  "$comment": "Generated from PoseInFrame by @foxglove/message-schemas",
+  "title": "PoseInFrame",
+  "description": "A timestamped pose in a named coordinate frame",
```

Comment:
"A timestamped position and orientation for an object or reference frame in 3D space"